Proxy/Proxy.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Net;
+
+namespace Proxy
+{
+    class Proxy : WebRequest
+    {
+        private readonly Request request;
+
+        public Proxy()
+        {
+            request = new Request();
```

Comment:
If you create new object in ctor - it's not a proxy

Reply:
done